GVFS/GVFS.Platform.Windows/ProjFSFilter.cs

```diff
@@ -64,7 +64,10 @@ public static bool TryAttach(ITracer tracer, string enlistmentRoot, out string e
                 if (result != OkResult && result != NameCollisionErrorResult)
                 {
                     errorMessage = string.Format("Attaching the filter driver resulted in: {0}", result);
-                    tracer.RelatedError(errorMessage);
+                    // This code is shared between GVFS.exe and GVFS.Service.exe
+                    // GVFS.exe failures will be retried by GVFS.Service.exe 
+                    // Warn the user and allow the caller to determine how to handle the error
+                    tracer.RelatedWarning(errorMessage);

```

Comment:
Though more to the point, I think we can just drop this line altogether. The callers already log all errors, so this is just duplicating the work. I think that applies to all the tracer calls in this method - I think it's not really this method's business to decide what is an error and what is not.

Reply:
@sanoursa makes sense thanks!  I was debating dropping the line, but thinking if we do it for one we'd want to drop them all.  Definitely cleaner this way.